test/local/account_routes.js

```diff
@@ -701,6 +701,20 @@ test('/account/login', function (t) {
       })
     })
 
+    t.test('location data is present in sign-in confirmation email', function (t) {
+      config.signinConfirmation.sample_rate = 1
```

Comment:
Nit: there is already a test setup that sets this to `1` (line 691), so these assertions should be merged into that one. Generally the aim in this module is to minimised duplication in test setup. So the tests are named according to what they do in setup, rather than what they assert.